modules/tinymce/src/core/test/ts/browser/table/InsertTableTest.ts

```diff
@@ -1,38 +1,49 @@
-import { describe, it } from '@ephox/bedrock-client';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
 import { TinyHooks } from '@ephox/wrap-mcagar';
 
 import Editor from 'tinymce/core/api/Editor';
 
 import * as TableTestUtils from '../../module/test/TableUtils';
 
 describe('browser.tinymce.core.table.InsertTableTest', () => {
-  const hook = TinyHooks.bddSetup<Editor>({
-    indent: false,
-    valid_styles: {
-      '*': 'width,height,vertical-align,text-align,float,border-color,' +
-           'background-color,border,padding,border-spacing,border-collapse'
-    },
-    base_url: '/project/tinymce/js/tinymce',
-    statusbar: false
-  }, []);
+  Arr.each([
+    { label: 'no colgroup', withColgroup: false },
+    { label: 'colgroup', withColgroup: true },
+  ], (scenario) => {
+    const { label, withColgroup } = scenario;
```

Comment:
This is sort of pointless if we're just destructing it immediately without any renaming or similar, so maybe something like this or just use `scenario.label`, etc... below instead?
```suggestion
  ], ({ label, withColgroup }) => {
```

Reply:
Done